src/renderer/assets/less/5_partials/modal.less

```
@@ -56,4 +69,11 @@
   &__action {
     margin: 0 0.5rem;
   }
+  &-button {
```

`__` is used for elements (http://getbem.com/naming/)
```suggestion
  &__button {
```

---

!!